Tempura/UITests/UITestCase.swift

```diff
@@ -49,6 +49,16 @@ public protocol UITestCase {
    */
   func uiTest(testCases: [String: V.VM], context: UITests.Context<V>)
   
+  /// Retrives a dictionary containing the scrollable subviews to test.
+  /// The snapshot will contain the whole scrollView content.
+  ///
+  /// - Parameters:
+  ///   - view: The `viewControllerModellableView` under test.
+  ///           `isViewReady` has already returned `true` at this point.
+  ///   - identifier: the test case identifier.
+  /// - Returns: A dictionary where the value is the ScrollView instance to snapshot and the key is a suffix for the test case identifier.
+  func scrollViewTestCases(in view: V, identifier: String) -> [String: UIScrollView]
```

Comment:
rename to `func scrollViewsToTest(in view: V, for identifier: String) -> [String: UIScrollView]`